apps/dashboard/src/app/App.tsx

```diff
@@ -52,11 +52,16 @@ export class App extends React.Component<IAppSetting, IAppState> {
         >
           {this.state.application === "interpret" && (
             <Interpret
-              dataset={
+              data={
```

Comment:
`data` is of type `IExplanationDashboardData`.

Technically, we could host `dataBalanceMeasures` within this `IExplanationDashboardData` interface.

However, there are comments such as this:
![image](https://user-images.githubusercontent.com/64443771/148107339-27ce9e0b-53da-4749-8e5a-e188e6cfbbd2.png)

Which leads us to host `dataBalanceMeasures` in `IDataset` and then pass this `IDataset` interface from parent to children (this file L60).